test/optimizer/operator_transformer_test.cpp

```diff
@@ -0,0 +1,838 @@
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+#include "binder/bind_node_visitor.h"
+#include "catalog/catalog.h"
+#include "optimizer/logical_operators.h"
+#include "optimizer/operator_expression.h"
+#include "optimizer/query_to_operator_transformer.h"
+#include "parser/expression/aggregate_expression.h"
+#include "parser/expression/column_value_expression.h"
+#include "parser/expression/operator_expression.h"
+#include "parser/expression/subquery_expression.h"
+#include "parser/postgresparser.h"
+#include "storage/garbage_collector.h"
+#include "traffic_cop/statement.h"
+#include "traffic_cop/traffic_cop.h"
+#include "transaction/deferred_action_manager.h"
+#include "transaction/transaction_manager.h"
+#include "util/data_table_benchmark_util.h"
+#include "util/test_harness.h"
+
+using std::make_tuple;
+
+using std::unique_ptr;
+using std::vector;
+
+namespace terrier {
+
+class OperatorTransformerTest : public TerrierTest {
```

Comment:
Switch all of the `LOG_INFO` statements to `LOG_DEBUG` to reduce the amount of output during normal execution.